react/features/toolbox/reducer.js

```diff
@@ -15,114 +15,81 @@ import {
-    if (alwaysVisible === true) {
-        visible = true;
-    }
-
-    return {
-        /**
-         * The indicator which determines whether the Toolbox should always be
-         * visible. When false, the toolbar will fade out after timeoutMS.
-         *
-         * @type {boolean}
-         */
-        alwaysVisible,
-
-        /**
-         * The indicator which determines whether the Toolbox is enabled.
-         *
-         * @type {boolean}
-         */
-        enabled: true,
-
-        /**
-         * The indicator which determines whether a Toolbar in the Toolbox is
-         * hovered.
-         *
-         * @type {boolean}
-         */
-        hovered: false,
-
-        /**
-         * The indicator which determines whether the overflow menu(s) are to be displayed as drawers.
-         *
-         * @type {boolean}
-         */
-        overflowDrawer: false,
-
-        /**
-         * The indicator which determines whether the OverflowMenu is visible.
-         *
-         * @type {boolean}
-         */
-        overflowMenuVisible: false,
-
-        /**
-         * A number, non-zero value which identifies the timer created by a call
-         * to setTimeout() with timeoutMS.
-         *
-         * @type {number|null}
-         */
-        timeoutID: null,
-
-        /**
-         * The delay in milliseconds before timeoutID executes (after its
-         * initialization).
-         *
-         * @type {number}
-         */
-        timeoutMS,
-
-        /**
-         * The indicator that determines whether the Toolbox is visible.
-         *
-         * @type {boolean}
-         */
-        visible
-    };
-}
+const INITIAL_STATE = {
+
+    /**
+     * The indicator which determines whether the Toolbox should always be
+     * visible. When false, the toolbar will fade out after timeoutMS.
+     *
+     * @type {boolean}
+     */
+    alwaysVisible: false,
+
+    /**
+     * The indicator which determines whether the Toolbox is enabled.
+     *
+     * @type {boolean}
+     */
+    enabled: true,
+
+    /**
+     * The indicator which determines whether a Toolbar in the Toolbox is
+     * hovered.
+     *
+     * @type {boolean}
+     */
+    hovered: false,
+
+    /**
+     * The indicator which determines whether the overflow menu(s) are to be displayed as drawers.
+     *
+     * @type {boolean}
+     */
+    overflowDrawer: false,
+
+    /**
+     * The indicator which determines whether the OverflowMenu is visible.
+     *
+     * @type {boolean}
+     */
+    overflowMenuVisible: false,
+
+    /**
+     * A number, non-zero value which identifies the timer created by a call
+     * to setTimeout() with timeoutMS.
+     *
+     * @type {number|null}
+     */
+    timeoutID: null,
+
+    /**
+     * The delay in milliseconds before timeoutID executes (after its
+     * initialization).
+     *
+     * @type {number}
+     */
+    timeoutMS: navigator.product === 'ReactNative'
+        ? TOOLBAR_TIMEOUT_MOBILE
```

Comment:
As mentioned above mobile no longer uses a timeout, so please remove that constant altogether.